scene/gui/shortcut.cpp

```diff
@@ -29,10 +29,10 @@
 /*************************************************************************/
 
 #include "shortcut.h"
-
 #include "core/os/keyboard.h"
 
 void Shortcut::set_event(const Ref<InputEvent> &p_event) {
+	ERR_FAIL_COND(Object::cast_to<InputEventShortcut>(*p_event));
```

Comment:
Try to avoid writing `ERR_FAIL_COND` without a message. Can you please add a message as to why this is not allowed? e.g.
`Cannot set set a shortcut event to an instance of InputEventShortcut`

@Bhu1-V

Reply:
Done! Now, could you be a pal give an approval to that PR? :stuck_out_tongue_closed_eyes: